docs/sitemap.md

```diff
@@ -0,0 +1,22 @@
+# Sitemap
+
+The `/sitemap.xml` URL provides the complete BigCommerce sitemap, which includes canonical URLs for all channels associated with Catalyst.
+These URLs correspond to the [permanent URLs](/docs/storefront/graphql#i-want-to-run-requests-in-the-context-of-another-channel) used when making requests to the GraphQL Storefront API. The URLs correspond to a specific channel, and when accessed, returns the relevant sitemap for that channel.
```

Comment:
Just confirming, are canonical URLs the same as permanent URLs? Are custom domains supported?

Reply:
Canonical does equal permanent URLs. 

>Are custom domains supported?

Not sure what you mean here? Maybe my previous comment answers this?

Reply:
Yeah, I think so 👍